### DISCUSSION TITLE
Dual R3 shutter relay

### DISCUSSION THREAD

**Discussion by glefloch** [arendst/Tasmota#16787] [Q&A] [answered]
Hi,

I'm a new tasmota user. I have flashed my dual r3 and plug it to shutter, this works like a charm. 
However, I can turn both relay on, which can be dangerous for the shutter motor. 
Is there a way to confiture tasmota to make sure this not happen? 

Thanks !

**Comment by barbudor** [comment, marked as answer]
How is you motor(s) connected to the Dual R3 ? 
Which mode have you configured ? 

There are different modes described in the [doc](https://tasmota.github.io/docs/Blinds-and-Shutters/#shutter-modes)
With 2 motors on a Dual, you are likely using ShutterMode 1

Did you used `Interlock` command ?

**Reply by glefloch** [reply]
I tried setting `cmnd/volet_2/Interlock 1 1,2` in order to enable interlock with one group containing both relay but it doesn't seem to change anything, am I wrong in my command ?

**Reply by glefloch** [reply]
Ok, I found the correct comment `cmnd/%topic%/Interlock ON`
